src/api/data_types/chunking/compression.rs

```diff
@@ -1,16 +1,18 @@
 use std::fmt;
 
-use serde::{Deserialize, Deserializer};
+use serde::Deserialize;
 
-#[derive(Debug, Clone, Copy, Ord, PartialOrd, Eq, PartialEq, Default)]
+#[derive(Debug, Clone, Copy, Ord, PartialOrd, Eq, PartialEq, Default, Deserialize)]
+#[serde(rename_all = "lowercase")]
 pub enum ChunkCompression {
-    /// No compression should be applied
-    #[default]
-    Uncompressed = 0,
     /// GZIP compression (including header)
     Gzip = 10,
     /// Brotli compression
     Brotli = 20,
+    /// No compression should be applied
+    #[default]
+    #[serde(other)]
```

Comment:
`#[serde(other)]` has to come last. I don't think that reordering this enum should have any effect, since we already manually set the enum values, and this PR keeps those the same